zhaquirks/tuya/ts004f.py

```diff
@@ -139,31 +139,31 @@ class TuyaSmartRemote004FROK(CustomDevice):
             COMMAND: COMMAND_MOVE_SATURATION,
             ENDPOINT_ID: 1,
             CLUSTER_ID: 768,
-            PARAMS: {"move_mode": 1, "rate": 200},
+            PARAMS: {"move_mode": 1},
         },
         (ROTATED_SLOW, RIGHT): {
```

Comment:
Rotating slow and fast is using different move rate.
If deleting the move rate it shall also deleting one of the slow/fast rotating  DAs then they its being the same functions.
DA with only rotating is using the "tuya scenes command" and the slow/fast is using light commands so need finding one new good name for this functions if deleting one of them for not getting DAs with the same names.